### ISSUE TITLE
[installation/docs] ESRI Add-in .esriAddinX does not always show up in Add-in list

### ISSUE THREAD

**Issue by catchSheep** [frontiersi/tenement-tools#4]
**Type of issue:**
Installation/documentation

**Issue:**
When following the install instructions for 2.9 or 3.1, adding the folders to the add-in manager folder search in the options tab  does not always show the add-in, even with "Load all Add-Ins without restrictions" and restarts. 
For me this is reproduced by:
- removing any packages and folders to search
- restart
- add folder to search
- restart
- package does not show up unless manually added

**Versions:**
ArcGIS Pro 2.9.10 with package v1.0.2 or 3.1.3 with package v1.0.4

**Potential solution:**
Add to the documentation to try to open the .esriAddinX file manually while ArcGIS Pro is open at [step 5](https://github.com/frontiersi/tenement-tools/blob/main/docs/install_arcgispro_31.md) of the installation process
Continue with steps as per normal, and installation succeeds

**Comment by lachlanhurst** [comment]
Thanks @catchSheep . I have experienced similar and in my case it came down to some odd behaviour in the ArcGIS Pro folder selection process.

The screen recording shows this. If I navigate into the unzipped tenement-tools folder and click ok the path becomes

    \\Mac\Home\Downloads\tenement-tools-1-0-4-arcgispro29\tenement-tools-1-0-4-arcgispro29

But if instead I select the folder from the level up and click ok I get

    \\Mac\Home\Downloads\tenement-tools-1-0-4-arcgispro29

The former folder does not exist, so obviously the plugin doesn't get detected.

Could this be the issue you are having?


https://github.com/frontiersi/tenement-tools/assets/4476041/25265a70-1e5c-46a0-ba6c-cd39bf36788a

**Comment by catchSheep** [comment]
Good catch, that does seem to have been the issue. Strange.

Might be good to make a note of it in the installation step until ESRI fixes that behavior.
Either that or perhaps instead a troubleshooting section at the end of the install docs, or in a separate file linked in the installation docs, that goes through common issues like this?

**Comment by lachlanhurst** [comment]
Agreed, we'll leave this open until we add a note into the install guide.

Longer term it'd be nice to have a much more streamlined install package that doesn't require users to go through so many steps.
